app/models/exercise_participation.rb

```diff
@@ -0,0 +1,25 @@
+class ExerciseParticipation
+  include Neo4j::ActiveRel
+  include Neo4j::Timestamps
+
+  from_class :Person
+  to_class :Exercise
+  type :participated_in
+
+  property :difficulty_rating
+  property :run_rating
```

Comment:
Relationships can have properties. So the student's rating is connected to the relationship between the Student and the Exercise.

The relationships and relationship properties would be accessed like this:

```ruby
Student.first.exercises_taken.each_with_rel do |exercise, participation|
 participation.fun_rating
 participation.difficulty_rating
end
```